oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoVersionGCSupport.java

```diff
@@ -258,7 +259,7 @@ public Iterable<NodeDocument> getModifiedDocs(final long fromModified, final lon
                         and(gt(MODIFIED_IN_SECS, getModifiedInSecs(fromModified)), lt(MODIFIED_IN_SECS, getModifiedInSecs(toModified)))));
```

Comment:
We need test coverage for this; the bug in the "current" code doesn't cause a test failure.

(working on it)